includes/Core/Assets/Assets.php

```diff
@@ -817,14 +787,15 @@ private function get_external_assets() {
 							'document.contains'                                      => $base_url . 'js/externals/wp-polyfill-node-contains.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
 							'window.FormData && window.FormData.prototype.keys'      => $base_url . 'js/externals/wp-polyfill-formdata.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
 							'Element.prototype.matches && Element.prototype.closest' => $base_url . 'js/externals/wp-polyfill-element-closest.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
+							'typeof URL === \'function\''                            => $base_url . 'js/externals/wp-polyfill-url.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
 						);
-						$polyfill_scripts = '/*googlesitekit*/';
+						$polyfill_scripts = '/*googlesitekit*/' . "\n";
```

Comment:
Just checking: this does not affect `Admin_Bar::add_amp_dev_mode`, right?

Reply:
Each test is done in the same inline script which `/*googlesitekit*/` is added to there, so this is still matched.

![image](https://user-images.githubusercontent.com/1621608/72723918-90dab780-3b8a-11ea-80e4-dc138af58147.png)

Should it be added to the injected script tags as well?

Reply:
No, that's not needed. All good then 👍